src/components/NavigationBar/index.js

```diff
@@ -633,13 +634,28 @@ class NavigationBar extends Component {
                         )}
                       </div>
                     </StyledIconButton>
-                    <IconButton
-                      color="inherit"
-                      onClick={() => history.push('/dashboard')}
-                      style={{ padding: '7px' }}
+                    <Tooltip
+                      placement="bottom"
+                      title="Dashboard"
+                      PopperProps={{
```

Comment:
Code is repetitive here. Please make a common component, so that we can use Tooltip in the following pattern.
```
<Tooltip text="Dashboard">
   <Children />
</Tooltip>

```